tests/app/forms/test_fields.py

```diff
@@ -9,7 +9,7 @@
 from app.data_model.answer_store import AnswerStore
 
 
-class TestFields(unittest.TestCase):
+class TestFields(unittest.TestCase):  # pylint: skip-file
```

Comment:
same as `test_mwss_submission`?

Reply:
These files are meant to be ignored as the changes suggested by lint to get the tests working cause more complexity and thus far less readable then whats already in place and working.

Reply:
What lint rules fail? I don't think we should be skipping files, could we just disable the lint rules that cause issues?